ps_menutoplinks.class.php

```diff
@@ -67,6 +67,16 @@ public static function getLinkLang($id_linksmenutop, $id_shop)
         return array('link' => $link, 'label' => $label, 'new_window' => $new_window);
     }
 
+    public static function updateUrl($link)
+    {
+        for($i = 1; $i <= count($link); $i++) {
+            if (substr($link[$i], 0, 7) !== "http://" && substr($link[$i], 0, 8) !== "https://") {
+                $link[$i] = "http://" . $link[$i];
```

Comment:
You had http, but if the server is using https, it could be not good. See if you can find some var on $_SERVER ? WDYT ?